ruby-gem/bin/calabash-android-generate.rb

```diff
@@ -1,20 +1,63 @@
-
-def calabash_scaffold
+def calabash_scaffold(questions = true)
   if File.exists?(@features_dir)
     puts "A features directory already exists. Stopping..."
     exit 1
   end
-  msg("Question") do
-    puts "I'm about to create a subdirectory called features."
-    puts "features will contain all your calabash tests."
-    puts "Please hit return to confirm that's what you want."
-  end
-  exit 2 unless STDIN.gets.chomp == ''
 
-  FileUtils.cp_r(@source_dir, @features_dir)
+  if questions
+    msg("Question") do
+      puts "I'm about to create a features directory here:"
+      puts ""
+      puts "#{ENV["PWD"]}/features"
```

Comment:
I don't know if ENV["PWD"] will always be set. We should use a Ruby method to obtain the current working directory.